utils/lib/classHelpers.ts

```diff
@@ -0,0 +1,24 @@
+import { GITHUB_REPO_BASE_URL } from '../constants';
+/**
+ * Creates the GitHub url of an asset within the main directory of a quickstart.
+ * @param assetFilePath - The file path of an asset.
+ * @return Returns an object containing the GitHub url of an asset.
+ */
+export const getAssetSourceUrl = (assetFilePath: string): string => {
+  // path = dashboards/apache/screenshot1.png
+  return `${GITHUB_REPO_BASE_URL}/${assetFilePath}`;
+};
+
+/**
+ * Removes the specified path prefix from a file path
+ * @param filePath - The path to change
+ * @param basePath - The portion of the path to remove
+ * @returns - The path with the prefix
+ */
+export const removeBasePath = (filePath: string, basePath: string): string => {
+  const shortPath = filePath.split(`${basePath}/`).pop();
+  if (typeof shortPath === 'string') {
+    return shortPath;
+  }
+  return filePath;
```

Comment:
Another way to handle this would be with a [short-curcuit `||` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR#short-circuit_evaluation), since `pop` is always going to return a string or a falsy value:
```ts
  return shortPath || filePath;
```
Another option would be the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator):
```suggestion
return shortPath ?? filePath
```
The second would keep the functionality the same

Reply:
Done!